src/main.rs

```diff
@@ -142,11 +142,11 @@ mod tests {
 		assert_eq!(None, parse_content(&"<title>    </title>".to_string()));
 		assert_eq!(None, parse_content(&"floofynips, not a real webpage".to_string()));
 		assert_eq!(Some("cheese is nice".to_string()), parse_content(&"<title>cheese is nice</title>".to_string()));
-		assert_eq!(Some("squanch".to_string()), parse_content(&"<title>     squanch</title>".to_string()));
```

Comment:
I don't think these need to be re-tabbed

Reply:
I just used `M - x tabify` in emacs, it's clearly doing something a little strange here.